src/platforms/php/guides/laravel/configuration/laravel-options.mdx

```diff
@@ -4,47 +4,48 @@ sidebar_order: 1
 description: "Learn about Sentry's integration with Laravel and its options for breadcrumbs, and performance."
 ---
 
-All configuration for Laravel is done in `config/sentry.php`.
+Most configuration for Laravel is done in your `.env` file.
+You can also add additonal config options to your `config/sentry.php` file.
 
 ## Breadcrumbs
 
-The Laravel integration will create [breadcrumbs](../../enriching-events/breadcrumbs/) for certain events occurring in the framework, the capture of this information can be configured using the following options:
+The Laravel SDK will create [breadcrumbs](../../enriching-events/breadcrumbs/) for certain events occurring in the framework, the capture of this information can be configured using the following environment variables:
 
 ```php {filename:config/sentry.php}
 'breadcrumbs' => [
```

Comment:
I love that you made the comments more clear here, super helpful.

Reply:
@stayallive Sharing the praise! 😅